tests/mapping/swagger20_request_validator/enum_validator_test.py

```diff
@@ -0,0 +1,30 @@
+import mock
+
+from bravado.mapping.swagger20_validator import enum_validator
+
+
+def test_multiple_jsonschem_call_if_enum_items_present_as_array():
+    enums = ['a1', 'b2', 'c3']
+    param_schema = {
+        'name': 'foo',
+        'in': 'query',
+        'type': 'array',
+        'items': {'type': 'array'},
+        'enum': enums
+        }
+    with mock.patch('jsonschema._validators.enum') as m:
+        [list(x) for x in enum_validator(
```

Comment:
`for` loop preferred - http://stackoverflow.com/questions/14249355/python-list-comprehension-without-assignment